test/WebSites/ActivatorWebSite/TagHelpers/HiddenTagHelper.cs

```diff
@@ -11,13 +11,24 @@ namespace ActivatorWebSite.TagHelpers
     [TargetElement("span")]
     public class HiddenTagHelper : TagHelper
     {
-        public string Name { get; set; }
+        public HiddenTagHelper(IHtmlHelper htmlHelper)
+        {
+            HtmlHelper = htmlHelper;
+        }
+
+        [HtmlAttributeNotBound]
```

Comment:
a) Had not seen this when I made comment above.
b) Don't need `[HtmlAttributeNotBound]` on any property that does not have both a `public` setter and a `public` getter.